himbaechel/himbaechel_dbgen/chip.py

```diff
@@ -166,6 +167,7 @@ def serialise_lists(self, context: str, bba: BBAWriter):
     def serialise(self, context: str, bba: BBAWriter):
         bba.u32(self.name.index)
         bba.u32(self.wire_type.index)
+        bba.u32(self.gfx_wire_id)
```

Comment:
this is a breaking change for existing binaries so it needs a db version bump:

https://github.com/YosysHQ/nextpnr/blob/9c2d96f86ed56b77c9c325041b67654f26308270/himbaechel/arch.cc#L36

and

https://github.com/YosysHQ/nextpnr/blob/9c2d96f86ed56b77c9c325041b67654f26308270/himbaechel/himbaechel_dbgen/chip.py#L834

Reply:
this needs to go to 5, 4 is the current version on main

Reply:
ok to rebase it  now, so it is easier then to spot things like this ?

Reply:
sounds like a good plan!